tasks.php

```diff
@@ -1562,11 +1562,7 @@ function NotificationMail($tid, $message, $new_task = false)
     $task_summary = $row['task_summary'];
 
     $subject = "$site_abbreviation Task #$tid: $task_summary";
-    $footer = "\n\n"
-        . "--\n"
-        . "$site_name\n"
-        . "$tasks_url?task_id=$tid\n\n"
-        . "This is an automated message, please do not respond directly to this e-mail.";
+    $footer  = "\n\n$tasks_url?task_id=$tid";
```

Comment:
Hm ... is this slightly different from the other emails? The old footer explicitly says "please do not respond directly to this e-mail" (hm ... `email` or `e-mail`?) Is it useful to have it there?

Reply:
No, I don't think so. I'd rather bring this into the fold with the other emails.